src/app/models/image-information.ts

```diff
@@ -0,0 +1,25 @@
+import { ListProp, Model, Prop, Record } from "@batch-flask/core";
+import { List } from "immutable";
+import { ImageReference, ImageReferenceAttributes } from "./image-reference";
+
+export interface ImageInformationAttributes {
+    verificationType: string;
```

Comment:
shouldn't this be an enum? or at least a union type of the available values

Reply:
Yea should be an enum